src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore/WrapPipelineMiddleware.cs

```diff
@@ -77,6 +87,11 @@ Task SetOutboundTracingDataAsync()
             }
         }
 
+        private string GetHeaderValue(IHeaderDictionary headers, string key)
+        {
+            return headers[key];
```

Comment:
Should we consider a case insensitive lookup? I remember seeing this raised in slack, but don't remember the outcome.

Reply:
The `IHeaderDictionary` does support case insensitive lookup so we are good there.

Reply:
I only know this because I was just looking at the Synthetics header code.. but It sounds like http headers 'should' be treated case insensitive per this spec:  https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

Reply:
Where I got schooled: 
https://github.com/newrelic/newrelic-dotnet-agent/blob/6271f650dd0e035633a2bc4ef36c272368e8b25a/src/Agent/NewRelic/Agent/Core/Transactions/Transaction.cs#L607-L608